docs/developer/release-process.md

```diff
@@ -165,21 +163,39 @@ For doing so, execute the following commands:
 export VERSION_NAME="v1.0.0-beta.1" # edit it accordingly
 
 git tag ${VERSION_NAME}
-git push origin ${VERSION_NAME}
+git push origin ${VERSION_NAME} # replace `origin` by your remote name
 ```
 
-A new tag pushed to the repository will trigger, apart from the usual test and build steps, a _release_ [workflow](https://circleci.com/gh/kubeapps/workflows) as described in the [CI documentation](./ci.md).
+> You can retrieve the `VERSION_NAME` using the [semver tool](https://github.com/fsaintjacques/semver-tool) for properly increasing the version from the latest pushed tag:
+>
+> ```bash
+> export VERSION_NAME="v$(semver bump <major|minor|patch> $(git fetch --tags && git describe --tags $(git rev-list --tags --max-count=1)))"
+> ```
 
-> When a new tag is detected, Bitnami will automatically build a set of container images based on the tagged commit. They later will be published in [the Dockerhub image registry](https://hub.docker.com/search?q=bitnami%2Fkubeapps&type=image).
+A new tag pushed to the repository will trigger, apart from the usual test and build steps, a _release_ [workflow](https://circleci.com/gh/kubeapps/workflows) as described in the [CI documentation](./ci.md). An example of the triggered workflow is depicted below:
 
-## 4 - Complete the GitHub release notes
+![CircleCI workflow after pushing a new tag](../img/ci-workflow-release.png "CircleCI workflow after pushing a new tag")
+
+> When a new tag is detected, Bitnami will automatically build a set of container images based on the tagged commit. They later will be published in [the Bitnami Dockerhub image registry](https://hub.docker.com/search?q=bitnami%2Fkubeapps&type=image).
+> Please note that this process is independent from the Kubeapps release process.
```

Comment:
It's not independent (we're triggering it), but it is run outside the control of the Kubeapps release process - is that what you mean?